src/Utilities/styleIndex.js

```diff
@@ -1,7 +1,84 @@
-export const backgroundColor = '#2D71A8';
+import React, { Component } from 'react';
+import { Header, Input, Button, Icon } from 'react-native-elements';
 
-export const lightAccent = '#53CFFF';
+export const palette = {
+  backgroundColor: '#2D71A8',
+  lightAccent: '#53CFFF',
+  darkAccent: '#104D80',
+  deepAccent: '#003562'
+};
 
-export const darkAccent = '#104D80';
+export const buttonDropShadow = {
+  shadowColor: "#000",
+  shadowOffset: { width: 0, height: 2, },
+  shadowOpacity: 0.23,
+  shadowRadius: 2.62,
+  // 
+  elevation: 4
+};
 
-export const deepAccent = '#003562';
\ No newline at end of file
+export const headerDropShadow = {
+  shadowColor: "#000",
+  shadowOffset: { width: 0, height: 6, },
+  shadowOpacity: 0.37,
+  shadowRadius: 7.49,
+  // 
+  elevation: 12
+}
+
+
+export const welcomeText = (name) => {
```

Comment:
Fantastic.  This is the essence of component-based code!  Very good work here, my friend!